suites/test_assembly_edit_publish.py

```diff
@@ -14,6 +14,7 @@
 sys.path.append("..")
 
 assembly_title_prefix = base.config_reader('test_repo', 'assembly_prefix')
+cp_url = base.config_reader('qa', 'cp_url')
```

Comment:
@noopurmath3 instead of hard coding this to 'qa', should we not make it to read the ENV variable, just in case the test is running against `prod` or 'stage`?